src/components/Boards/BoardsContainer.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Boards from './Boards';
+import useGroups from '../../lib/hooks/useGroups';
+
+export default function BoardsContainer() {
+  const { data, isLoading, error } = useGroups();
+
+  if (isLoading) {
+    return <div>Loading...</div>;
```

Comment:
We can render loading spinner from material-ui now. We can do this in next PR if you want

Reply:
This is just a placeholder for now yea. Let's polish UI up later.

Reply:
Sounds good